datafusion/expr/src/expr.rs

```diff
@@ -840,6 +840,12 @@ impl WindowFunction {
     }
 }
 
+/// Find DataFusion's built-in window function by name.
+#[deprecated(since = "44.0.0", note = "built-in window functions have been removed")]
+pub fn find_df_window_func(_name: &str) -> Option<WindowFunctionDefinition> {
+    None
```

Comment:
As someone who did upgrades, I'd appreciate a compiler error more than a dummy implementation.

But we can do good here. We can see what's the name being looked up and return non-built-in implementation based on that. The return type allows to do this, fortunately.

Reply:
That makes sense. I am moving this to draft until I can implement the feedback.